src/components/Card.jsx

```diff
@@ -0,0 +1,52 @@
+import "./Card.css";
+
+const displayCreatedAt = (createdAt) => {
+  const date = new Date(createdAt);
+  const now = Date.now();
+  const milliSeconds = now - date;
+
+  const seconds = milliSeconds / 1000;
+  const minutes = seconds / 60;
+  const hours = minutes / 60;
+  const days = hours / 24;
+  const months = days / 30;
+  const years = months / 12;
+
+  if (seconds < 60) {
+    return "방금 전";
+  } else if (minutes < 60) {
+    return `${Math.floor(minutes)}분 전`;
+  } else if (hours < 24) {
+    return `${Math.floor(hours)}시간 전`;
+  } else if (days < 30) {
+    return `${Math.floor(days)}일 전`;
+  } else if (months < 12) {
+    return `${Math.floor(months)}달 전`;
+  } else {
+    return `${Math.floor(years)}년 전`;
+  }
+};
+
+const Card = ({ data }) => {
+  const { createdAt, description, imageSource, url } = data;
+  if (!data) return;
+
+  let time = displayCreatedAt(createdAt);
+
+  return (
+    <div className='Card'>
+      <a href={url} target='_blank' rel='noreferrer'>
+        <div className='card-image'>
+          <img src={imageSource} alt='imageSource' />
+        </div>
+        <div className='description'>
+          <span>{time}</span>
+          <p>{description}</p>
+          <span>{new Date(createdAt).toLocaleDateString()}</span>
```

Comment:
렌더부에서 연산을 해주면 계속 재연산되어서 좋지 않아요!
위에서 변수로 선언해주고, 렌더부에서는 갖다 쓰시기만 하는 걸 추천드립니다
```suggestion
         // 위쪽
         const displayDate = new Date(createdAt).toLocaleDateString()
         // 아래 렌더부
          <span>{displayDate}</span>
```